### ISSUE TITLE
[Bug] "LoRA Loader (Block Weight)" node with ALL Flux blocks still gives a different image if the LoRa has the Text Encoder trained 

### ISSUE THREAD

**Issue by diodiogod** [ltdrdata/ComfyUI-Inspire-Pack#185]
As the title says. If the LoRa has been trained with TE, the result with all blocks at 1 is different and it should be the same as loading it with the other LoRa Loaders. If the LoRa had NO TE trained it behaves normally.
Only if I lower the clip strength to 0, then the result is the same as the other loaders. there is probably some kind of calculation with the TE for the " LoRA Loader (Block Weight) " node that should not have. 
Maybe I'm setting something wrong. But I don't think so.
![ComfyUI_00233_](https://github.com/user-attachments/assets/596da994-99e9-4317-8fa7-482bb36143b3)

.
I can also confirm this does not happen on @nihedon implementation on Forge https://github.com/nihedon/sd-webui-lora-block-weight
![xyz_grid-0000-1102904567](https://github.com/user-attachments/assets/170c2cb4-3ce4-4bfd-8e4e-448756d152b7)

**Comment by diodiogod** [comment]
Here is a LoRA to test it: https://drive.google.com/file/d/1ENdXU3Gq0QadZkdpgg_ep436OTWlRNQ2/view?usp=sharing
A simple workflow to test it: https://drive.google.com/file/d/1sTTyZzJmcx2grbsUgK_YJaJpWtd7xa-4/view?usp=sharing

**Comment by ltdrdata** [comment]
I cannot reproduce the issue.

![same2](https://github.com/user-attachments/assets/e1f61a7e-9397-456b-8dea-d2a2e28dae8e)
![same1](https://github.com/user-attachments/assets/1013c490-dee7-49ce-9946-ae205d4ce70e)

Make sure your Inspire Pack is up to date.
(V1.6)

**Comment by diodiogod** [comment]
> 
> Make sure your Inspire Pack is up to date. (V1.6)

I updated now, but I have the same issue. 
I actually changed my 15xxl to the full version instead of the quantized one and got very similar images as your. The thing is, looking again at your screenshots, they are different! On your second image, look at the pockets, or the opening on bottom of the raincoat. It is not the same image as the others.
